test/reactive-handler.spec.ts

```diff
@@ -662,6 +662,52 @@ describe('ReactiveHandler', () => {
             expect(proxyDesc.get).toEqual(proxyCopiedDesc.get);
             expect(proxyDesc.set).toEqual(proxyCopiedDesc.set);
         });
+        it('should protect against leaking accessors into the blue side', () => {
+            const target = new ReactiveMembrane();
+            const todos = {};
```

Comment:
Declare the todos as any and you can avoid the rest of the typecast for todos below.
```suggestion
            const todos: any = {};
```